python_modules/dagster-test/dagster_test/graph_job_op_toys/asset_lineage.py

```diff
@@ -177,5 +175,3 @@ def asset_lineage():
 
 
 asset_lineage_job = asset_lineage.to_job(resource_defs={"my_db_io_manager": my_db_io_manager})
-
-warnings.resetwarnings()
```

Comment:
1. it took me way to long to find this which was causing the root ignore not to work 
2. this does not do what you think it would do - it doesn't set it back to default state it sets it to blank slate